Sources/CreateAPI/Generator/Generator+Schemas.swift

```diff
@@ -100,11 +100,25 @@ extension Generator {
     }
     
     private func shouldGenerate(name: String) -> Bool {
-        if !options.entities.include.isEmpty {
-            return options.paths.include.contains(name)
+        var entitiesToInclude: Set<String> {
+            if let placeholder = options.entities.namePlaceholder {
+                return Set(options.entities.include.map { placeholder.replacingOccurrences(of: "*", with: $0) })
```

Comment:
Please remove duplication. I think a good place for name generation is the existing `getTypeName` method.